CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+We welcome contributions! To contribute to this repository, please read these instructions:
+
+- [Project Organization](#project-organization)
+- [Code and Testing](#code-and-testing)
+- [Documentation and Maintenance](#documentation-and-maintenance)
+- [Development and Local Testing](#development-and-local-testing)
+
+---
+
+## Project Organization
+
+- **Project Configuration:** The setup for both production and development dependencies is outlined in the `pyproject.toml` file. This centralized approach simplifies dependency management.
+
+- **Automatic `requirements.txt`:** Changes to dependencies are automatically reflected in the `requirements.txt` file, derived from `pyproject.toml`. Direct modifications should be avoided; instead, update `pyproject.toml`.
```

Comment:
Better to address here that the auto-generated requirements.txt is used in creating docker images.